compilers/concrete-compiler/compiler/lib/Runtime/simulation.cpp

```diff
@@ -183,16 +205,147 @@ void sim_wop_pbs_crt(
 
 uint64_t sim_neg_lwe_u64(uint64_t plaintext) { return ~plaintext + 1; }
 
+uint64_t sim_add_lwe_u64(uint64_t lhs, uint64_t rhs, char *loc,
+                         bool is_signed) {
+  const char msg_f[] =
```

Comment:
Maybe a better error message could be
```
%loc: %lhs + %rhs overflow/underflow between unsigned/signed %bitwidth integers
```

I known we don't have bitwidth here so for the moment could be ok to not have the info, but I feel giving values could help a lot.

Reply:
without bitwidth we can't print the value, except if you mean encoded values (including the noise) which wouldn't make a lot of sense for the user IMO